test/functional/feature_sporks.py

```diff
@@ -1,63 +0,0 @@
-#!/usr/bin/env python3
-# Copyright (c) 2018-2023 The Dash Core developers
-# Distributed under the MIT software license, see the accompanying
-# file COPYING or http://www.opensource.org/licenses/mit-license.php.
-
-from test_framework.test_framework import BitcoinTestFramework
-
-'''
-'''
-
-class SporkTest(BitcoinTestFramework):
```

Comment:
so far as we do not remove Spork functionality completely, but still uses them on regtest and devnets, we should not remove this functional tests, even if that's a trivial one

Reply:
Agreed